packages/browser/src/__tests__/cookieless.test.ts

```diff
@@ -310,5 +310,32 @@ describe('cookieless', () => {
             expect(beforeSendMock.mock.calls[3][0].properties.$cookieless_mode).toEqual(true)
             expect(posthog.sessionRecording).toBeFalsy()
         })
+
+        it('should restart autocapture after opt_in_capturing in cookieless mode', async () => {
```

Comment:
I'm confused about whether the bug is related to autocapture not being restarted on opt-in (which is what this is testing), or whether it's related to the request queue not starting (which is the title of this PR).

Is there a link to a ticket somewhere?